src/constraint/spec.ts

```diff
@@ -264,6 +265,24 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
     }
   },
   {
+    name: 'omitAggregatePlotWithoutDimension',
+    description: 'Aggregate plots without dimension should be omitted',
+    properties: [Property.AGGREGATE, Property.AUTOCOUNT, Property.TYPE],
```

Comment:
So everytime you're not sure about something you should investigate, not just guess.  
I almost didn't catch this.   Don't expect that I'll catch every single line.